### ISSUE TITLE
As for the "SafetyAntRun-v0", how can I decrease?

### ISSUE THREAD

**Issue by Amanda2024** [SvenGronauer/Bullet-Safety-Gym#1]
As for the "SafetyAntRun-v0", how can I  reduce the width of the avenue?
And how can I change the reward structure to the agent receiving a reward only when it reaches the goal state (after successfully going through the avenue) ?

**Comment by SvenGronauer** [comment]
Dear Amanda,

1) The spawning position can be changed with self.y_lim which is currently set to 2 meters.
(https://github.com/SvenGronauer/Bullet-Safety-Gym/blob/master/bullet_safety_gym/envs/tasks.py#L471)

2) The reward function is implemented in the "calculate_reward" method (https://github.com/SvenGronauer/Bullet-Safety-Gym/blob/master/bullet_safety_gym/envs/tasks.py#L497). Set the reward to one one when the agent is close to the goal and zero otherwise. Please be aware that the current goal xy position [1000, 0] may be too far for an agent to reached within 1000 time steps.

**Comment by Amanda2024** [comment]
Sorry for the late reply, but I find the ant is almost no moving when the time steps reached 1000. What I used is a PPO agent. Can you explain it to me ?

**Comment by SvenGronauer** [comment]
Dear Amanda,

I ran TRPO on the "SafetyAntRun-v0" task for 500 epochs with a batch size of 64000 which worked out for me quite well (see: https://github.com/SvenGronauer/Bullet-Safety-Gym/blob/master/docs/benchmark.md).
There might be many reasons for such failure... I recommend to use the default hyper-parameters of PPO and at least 1M time steps of training.
